### PR TITLE
add jupyterlab-git extension

### PR DESCRIPTION
**Pull request by romeokienzler**
## Describe your changes
add jupyterlab-git extension

New size of the image: 3.82GB

## Issue ticket if applicable

 Fix: #1464 

## Checklist (especially for first-time contributors)

- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [x] I will try not to use force-push to make the review process easier for reviewers
- [ ] I have updated the documentation for significant changes

### CODE REVIEW

File: scipy-notebook/Dockerfile [hunk 1]
```diff
@@ -57,7 +57,8 @@ RUN mamba install --yes \
     'statsmodels' \
     'sympy' \
     'widgetsnbextension'\
-    'xlrd' && \
+    'xlrd'\
+    'jupyterlab-git' && \
```
Comment (mathbunnyru) [inline scipy-notebook/Dockerfile:61]: Let's keep this list sorted.

**Review by mathbunnyru — CHANGES_REQUESTED** [review]

**Review by mathbunnyru — CHANGES_REQUESTED** [review]
You will also need to update the docs here: https://github.com/jupyter/docker-stacks/blob/main/docs/using/selecting.md#jupyterscipy-notebook

File: scipy-notebook/Dockerfile (hunk 1 above)
Comment (romeokienzler) [reply scipy-notebook/Dockerfile]: @mathbunnyru pushed

**Comment by romeokienzler** [comment]
@mathbunnyru added documentation

**Comment by mathbunnyru** [comment]
@romeokienzler great 👍 
Please, do not forget to add an exception around here, because `jupyterlab-git` is not importable: https://github.com/jupyter/docker-stacks/blob/main/tests/base-notebook/test_packages.py#L72
This will fix CI tests.
You can also get familiar with how our tests work [here](https://jupyter-docker-stacks.readthedocs.io/en/latest/contributing/tests.html)

**Review by mathbunnyru — CHANGES_REQUESTED** [review]
Waiting for tests fix.

**Comment by romeokienzler** [comment]
Thanks @mathbunnyru, done

**Review by mathbunnyru — APPROVED** [review]
As this seems to be really popular package for jupyterlab, I think it makes sense to add it, so I'm ok with this change.

**Pull request merged**